### PR TITLE
Add support for serverless.template.yml

### PR DESCRIPTION
**Pull request by timqian**
## What has been implemented?

- Support publishing templates using `serverless.template.yml`
- Ignore `serverless.template.yml`
- Backward compatibility (support publishing template using `serverless.yml`)
- Update Chinese README
- Test in China

## TODO

- [ ] English README
- [ ] Test outside China

## Related issue

https://github.com/serverless/roadmap-tencent/issues/569

### CODE REVIEW

File: src/cli/commands-cn/registry.js [hunk 1]
```diff
@@ -23,36 +23,50 @@ const publish = async (config, cli) => {
...
+  // We want to check the existence of serverless.template.yml and serverless.component.yml first
+  // If both of them did not show up, we will check serverless.yml for backward compatibility
+  // Why not check the existence of serverless.yml first? serverless.template.yml and serverless.yml may be in the same folder
+  const serverlessTemplateFile = await loadTemplateConfig(process.cwd());
+  const serverlessComponentFile = await loadComponentConfig(process.cwd());
+  const serverlessFile = await loadServerlessFile(process.cwd());
+
+  if (!serverlessTemplateFile && !serverlessComponentFile && !serverlessFile) {
+    throw new Error(
+      'Publish failed. The current working directory does not contain a "serverless.template.yml" or "serverless.component.yml" or "serverless.yml"'
```
Comment (eahefnawy) [inline src/cli/commands-cn/registry.js:35]: I would remove `serverless.yml` from that error message completely.

File: src/cli/commands/registry.js [hunk 2]
```diff
@@ -42,56 +47,55 @@ const publish = async (config, cli) => {
...
-      );
-    }
-
-    serverlessFile = serverlessComponentFile;
-    serverlessFile.src = serverlessComponentFile.src || serverlessComponentFile.main;
-  } else if (serverlessFile.version || serverlessFile.type === 'component') {
+  if (!serverlessTemplateFile && !serverlessComponentFile && !serverlessFile) {
     throw new Error(
-      'Publish failed. Components could only be defined with a "serverless.component.yml" file.'
+      'Publish failed. The current working directory does not contain a "serverless.yml" or "serverless.component.yml"'
```
Comment (eahefnawy) [inline src/cli/commands/registry.js:56]: same here

**Review by eahefnawy — DISMISSED** [review]
Looks good. Just had some minor comments. I haven't tested it though. Have you tested all the edge cases we discussed on the call?

File: src/cli/commands-cn/registry.js (hunk 1 above)
Comment (timqian) [reply src/cli/commands-cn/registry.js:35]: Yeah, removing it should be better

**Comment by timqian** [comment]
> Looks good. Just had some minor comments. I haven't tested it though. Have you tested all the edge cases we discussed on the call?

I have tested in China for the following cases.
- [x] `serverless.template.yml` in the parent folder of the actual template code
- [x] `serverless.template.yml` and `serverless.yml` in the same folder and ignoring itself
- [x] `serverless.yml` in the parent folder of the actual template code

Do you think there are cases I missed?

**Review by eahefnawy — APPROVED** [review]

**Pull request merged**